File: src/core/__tests__/fixtures/sample.java
import flowconsole.sdk.*;
import java.util.List;

User user = new User(UserArgs.builder()
  .name("user")
  .description("Administrator user")
  .tags(List.of("admin", "user"))
  .badge("gold")
  .build());

ReactApp app = new ReactApp(ComponentArgs.builder()
  .name("app")
  .build());

RestApi api = new RestApi(ComponentArgs.builder()
  .name("api")
  .belongsTo(app)
  .icon("api-icon")
  .build());

user.sendsRequest(app, "Load App");
app.then(api);

File: src/core/__tests__/fixtures/sample.go
package main

import . "github.com/slackmaster9999/flowconsole"

user := NewUser(&UserArgs{
    Name:        "user",
    Description: "Administrator user",
    Tags:        []string{"admin", "user"},
    Badge:       "gold",
})

app := NewReactApp(&ReactAppArgs{
    Name: "app",
})

api := NewRestApi(&RestApiArgs{
    Name: "api",
})

user.SendsRequest(app, "Load App")
app.Then(api)
